templates/event-articles/py_events.njk

```diff
@@ -35,6 +35,14 @@
 			"MASK's Open Campus Anime Quiz was back, and it did not disappoint! The atmosphere crackled with energy as questions flew back and forth, with each answer met by cheers and groans.",
 			'ocaq-2023'
 		) }}
+		{{ event (
```

Comment:
move this to the top before OCAQ 2023, these will be displayed by the order you arrange here, and we want the lates ones first, so...